conf/cortx_rgw.conf

```diff
@@ -8,6 +8,7 @@
     rgw motr tm recv queue min len = 16
     motr tracing enabled = true
     rgw enable apis = s3, s3website, swift, swift_auth, admin, sts, iam, notifications
+    # Enabling cache causes data retrieval inconsistencies for get-object, head-object etc. ref <CORTX-31109>
```

Comment:
move this cfg to line 21 and comment with one blank line above it

And again, plz confirm that this comment wont be seen in deployed file as conf store would remove it.

Reply:
>> move this cfg to line 21 and comment with one blank line above it
Done. 

>> And again, plz confirm that this comment wont be seen in deployed file as conf store would remove it.
Yes, comments are not present in the deployed file. Below is the config file content after deployment.
```
cat /etc/cortx/rgw/e884506cadb150157e923ad8e4873ac4/cortx_rgw.conf
[global]
debug rgw = 20

[client]
rgw backend store = motr
rgw motr is oostore = true
rgw motr is read verify = false
rgw motr tm recv queue min len = 16
motr tracing enabled = true
rgw enable apis = s3, s3website, swift, swift_auth, admin, sts, iam, notifications
rgw thread pool size = 10
rgw max concurrent request = 1024
rgw init timeout = 300
rgw gc max objs = 32
rgw gc obj min wait = 1800
rgw gc processor max time = 3600
rgw gc processor period = 3600
rgw data path = /var/lib/ceph/radosgw/f7be9c7a30904bbf954e8de098789f4d
rgw cache enabled = false
motr layout id = 9
motr unit size = 1048576
motr max units per request = 8
motr max idx fetch count = 30
motr max rpc msg size = 524288
motr reconnect interval = 4
motr reconnect retry count = 15

[client.radosgw-admin]
motr profile fid = 0x7000000000000001:0x4f
motr ha endpoint = inet:tcp:10.230.245.117@22001
motr my endpoint = inet:tcp:10.230.245.117@21501
motr my fid = 0x7200000000000001:0x29
motr admin fid = 0x7200000000000001:0x29
motr admin endpoint = inet:tcp:10.230.245.117@21501
log file = /etc/cortx/log/rgw/e884506cadb150157e923ad8e4873ac4/radosgw-admin.log

[client.rgw-1]
motr profile fid = 0x7000000000000001:0x4f
motr ha endpoint = inet:tcp:10.230.245.117@22001
motr my endpoint = inet:tcp:10.230.245.117@21501
motr my fid = 0x7200000000000001:0x29
rgw frontends = beast port=22751 ssl_port=23001 ssl_certificate=/etc/cortx/solution/ssl/s3.seagate.com.pem ssl_private_key=/etc/cortx/solution/ssl/s3.seagate.com.pem
log file = /etc/cortx/log/rgw/e884506cadb150157e923ad8e4873ac4/rgw-1.log

[release]
version = 2.0.0-5514

```